sites/partners/src/listings/PaperListingForm/sections/ApplicationDates.tsx

```diff
@@ -1,15 +1,77 @@
-import React from "react"
-import { useFormContext, useWatch } from "react-hook-form"
-import { t, GridSection, DateField, TimeField } from "@bloom-housing/ui-components"
-import { FormListing } from "../index"
+import React, { useState, useMemo } from "react"
+import { useWatch, useFormContext } from "react-hook-form"
 import { YesNoAnswer } from "../../../applications/PaperApplicationForm/FormTypes"
+import { getDetailFieldDate, getDetailFieldTime } from "../../PaperListingDetails/sections/helpers"
 import moment from "moment"
 
+import {
+  t,
+  GridSection,
+  DateField,
+  TimeField,
+  Drawer,
+  Button,
+  AppearanceSizeType,
+  MinimalTable,
+  Modal,
+  AppearanceStyleType,
+  AppearanceBorderType,
+} from "@bloom-housing/ui-components"
+import { FormListing, TempEvent } from "../index"
+import { OpenHouseForm } from "../OpenHouseForm"
+
 type ApplicationDatesProps = {
+  openHouseEvents: TempEvent[]
+  setOpenHouseEvents: (events: TempEvent[]) => void
   listing?: FormListing
 }
 
-const ApplicationDates = ({ listing }: ApplicationDatesProps) => {
+const ApplicationDates = ({
+  listing,
+  openHouseEvents,
+  setOpenHouseEvents,
+}: ApplicationDatesProps) => {
+  const openHouseHeaders = {
+    date: t("t.date"),
```

Comment:
I think these are passed as keys ie "t.date" and not translated

Reply:
hmm, but it's a proper translation path, I don't have any errors, and strings are translated:

![Partners Portal 2021-07-28 at 5 31 07 PM](https://user-images.githubusercontent.com/16655908/127351152-fe0afb00-c07b-4b7d-a9af-8e9f41993168.jpg)

Reply:
I am getting console errors about missing keys I believe due to this issue?